.github/workflows/acceptance-tests.yaml

```diff
@@ -15,6 +15,7 @@ jobs:
           - Container
           - Domain
           - Function
+          - Iam
```

Comment:
Does this enable nightly tests?

Reply:
This does not enable nightly tests, there is a product list for that